content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -61,6 +61,19 @@ among the comma-delimited, ordered list of values for the `--enable-admission-pl
 the API server component. For more information on API server command-line flags,
 check [kube-apiserver](/docs/admin/kube-apiserver/) documentation.
 
+### Retroactive StorageClass assignment
```

Comment:
I'd like to move this much later down the page - perhaps just before “Volume populators and data sources”.

Someone who is trying to learn about PVs and PVCs work is going to find this detail is mentioned too early in their learning journey.

Reply:
Done.